build.sbt

```diff
@@ -29,6 +30,9 @@ lazy val sslConfigCore = project.in(file("ssl-config-core"))
       "0.3.8",
       "0.4.0",
     ))).map("com.typesafe" %% "ssl-config-core" % _), // "sbt mimaReportBinaryIssues"
+    mimaBinaryIssueFilters ++= Seq(
+      ProblemFilters.exclude[IncompatibleSignatureProblem]("com.typesafe.sslconfig.ssl.AlgorithmConstraintsParser.*")
```

Comment:
Hmm, hard to say if this is an actual problem, see also https://github.com/lightbend/ssl-config/pull/188#issuecomment-512705775

Reply:
Sounds like the combination of static forwarders changes and/or raw types.

I'm happy to assume it'll be fine.